plumpy/persistence.py

```diff
@@ -27,9 +27,9 @@
 class Bundle(dict):
 
     @classmethod
-    def from_dict(cls, *args, **kwargs):
-        self = Bundle.__new__(*args, **kwargs)
-        super(Bundle, self).from_dict(*args, **kwargs)
+    def from_dict(cls, **kwargs):
```

Comment:
This method is not being used anywhere and to be honest I don't see the point. So I prefer to get rid of it. (That means you can also remove the test you added for it).

Reply:
Removed, thanks for reviewing.